sources/packages/web/src/components/students/modals/PDStatusApplicationModal.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <ModalDialogBase
+    title="Apply for a permanent disability status"
+    :showDialog="showModal"
+  >
+    <template v-slot:content>
+      <v-container>
+        <p>
+          If you’re eligible, you may receive funding for provincial and federal
+          grants/bursaries. Your information will be sent to Assistive
+          Technology BC (ATBC).
+        </p>
+      </v-container>
+    </template>
+    <template v-slot:footer>
+      <v-btn outlined color="primary" @click="hidePDApplicationModal()"
+        >No</v-btn
+      >
+      <v-btn color="primary" @click="applyPDStatus()">Yes</v-btn>
+    </template>
+  </ModalDialogBase>
+</template>
+
+<script lang="ts">
+import ModalDialogBase from "@/components/generic/ModalDialogBase.vue";
+
+export default {
+  components: {
+    ModalDialogBase,
+  },
+  props: {
+    showModal: {
+      type: Boolean,
+      required: true,
+    },
+  },
+  setup(props: any, context: any) {
```

Comment:
The code to control the dialog can be simplified here and on the consumer. We can talk about it.
Please see below how the code would looks like in the modal.
![image](https://user-images.githubusercontent.com/61259237/156278151-11014764-24f9-462f-8164-dd7af7cfe1f6.png)

And what the code will look like in the consumer.
```
 <PDStatusApplicationModal ref="pdStatusApplicationModal" />
```
![image](https://user-images.githubusercontent.com/61259237/156278242-06db934a-e48e-4f59-944c-6f79b94d53df.png)